### ISSUE TITLE
OMG,Epoch:4000

### ISSUE THREAD

**Issue by peylnog** [HongguangZhang/DMPHN-cvpr19-master#24]
Does it really need such a long time for reproduction ？OMG，this is the first time I've seen so much epochs.

**Comment by HongguangZhang** [comment]
No need. If your computational resources allow, you can set a large batchsize and initial learning rate to significantly reduce the training epochs.
